SocialFilter/sections/specific.txt

```diff
@@ -57,6 +57,9 @@ polaris-official.com,apexheadline.com,aucfan.com,sist8.com,spaceshipearth.jp,sou
 !
+7review.com##.social_icon > span[data-service="facebook"]
+7review.com##.social_icon > span[data-service="twitter"]
+7review.com##.social_icon > span[data-service="pinterest"]
```

Comment:
You can block it completely, because all the elements can be count as share ones.
For instance,
```
7review.com##.top_share
```

Reply:
Btw, if there was some functional buttons, it would be an exception.

Reply:
> You can block it completely, because all the elements can be count as share ones.

I know that. however, if `.top_share` is blocked, a visitor cannot use the email share button that does not rely on a third-party social media service.

Reply:
We had discussed the case and decided to block such elements as well- they are the part of the share element.